### PR TITLE
Feature/frondend in the making

### PR DESCRIPTION
**Pull request by sinejespersen**

### CODE REVIEW

File: assets/js/Steps/components/bannerStatus.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const bannerStatus = {
```
Comment (sinejespersen) [inline assets/js/Steps/components/bannerStatus.js:1]: this should be fixed...

File: assets/js/Steps/components/helpBox.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { faQuestionCircle } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+function HelpBox({ text }) {
```
Comment (tuj) [inline assets/js/Steps/components/helpBox.js:6]: Missing documentation

File: assets/js/Steps/components/iconBubble.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import MachineStateContext from "../../context/machineStateContext";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faBookReader,
+  faInfoCircle,
+  faBook,
+  faSignInAlt,
+} from "@fortawesome/free-solid-svg-icons";
+
+function IconBubble() {
```
Comment (tuj) [inline assets/js/Steps/components/iconBubble.js:11]: Missing documentation

File: assets/js/Steps/components/iconBubble.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  const context = useContext(MachineStateContext);
+  function renderStep() {
```
Comment (tuj) [inline assets/js/Steps/components/iconBubble.js:13]: new line and comment for function

File: assets/js/Steps/loginComponents/scanLogin.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    }, [actionHandler]);
+
+    return (
+        <>
+            <div className="col-md-9">
+                <Header header="Login" text="Scan låner stregkode"></Header>
+                <div className="row">
+                    <div className="col-md-2"></div>
+                    <div className="col-md mt-4">
+                        <div className="content" onClick={() => actionHandler('login', { username: 'C023648674', password: '' })}>
```
Comment (tuj) [inline assets/js/Steps/loginComponents/scanLogin.js:53]: remove hardcoded username

File: assets/js/Steps/loginComponents/uniLogin.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        <>
+            <div className="col-md-9">
+                <Header header="Login" text="Login med Unilogin"></Header>
+                <div className="row">
+                    <div className="col-md-2"></div>
+                    <div className="col-md mt-4">
+
+                        <div
+                            className="content"
+                            onClick={() => actionHandler('login', context)}
```
Comment (tuj) [inline assets/js/Steps/loginComponents/uniLogin.js:20]: Why pass all of machine state to actionHandler?

File: engine/package.json [hunk 7]
```diff
@@ -38,7 +38,7 @@
         "check-coding-standards-eslint": "eslint --config .eslintrc.js '**/*.js'",
         "check-coding-standards-js": "npm run check-coding-standards-eslint",
         "check-coding-standards": "npm run check-coding-standards-js",
-        "apply-coding-standards-eslint": "eslint --config .eslintrc.js '**/*.js' --fix",
+        "apply-coding-standards-eslint": "eslint --config .eslintrc.js '**/*.jsx' --fix",
```
Comment (tuj) [inline engine/package.json:41]: There are no jsx files in the engine

**Pull request merged**